examples/real-world/yarn.lock

```diff
@@ -0,0 +1,3805 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
this file should be removed, while you are at it you could add `yarn.lock` to `.gitignore`d files